docs/csharp/programming-guide/concepts/threading/index.md

```diff
@@ -1,34 +0,0 @@
----
-title: "Threading (C#)"
-ms.date: 07/20/2015
-ms.assetid: 236d157d-37c0-4ee8-89fc-721e6c596325
----
-# Threading (C#)
-Threading enables your C# program to perform concurrent processing so that you can do more than one operation at a time. For example, you can use threading to monitor input from the user, perform background tasks, and handle simultaneous streams of input.  
-  
- Threads have the following properties:  
-  
--   Threads enable your program to perform concurrent processing.  
-  
--   The .NET Framework <xref:System.Threading> namespace makes using threads easier.  
-  
--   Threads share the application's resources. For more information, see [Using Threads and Threading](../../../../../docs/standard/threading/using-threads-and-threading.md).  
-  
- By default, a C# program has one thread. However, auxiliary threads can be created and used to execute code in parallel with the primary thread. These threads are often called *worker threads*.  
```

Comment:
It seems to me that this and the next paragraph provide useful context but do not seem to appear in the overview at https://docs.microsoft.com/dotnet/standard/threading/threads-and-threading. I think that that topic could benefit from having these two paragraphs added, and also could be broken up into sections. Currently, there's only "When to use multiple threads". This could go first ("Threads and worker threads"), followed by "Threads, application domains, and processes", followed by the multiple threads topic. 

A bit of rewording and rewriting would also be needed to make the topic flow smoothly.

Reply:
@rpetrusha I agree, I'll use the opportunity and take a look on how to improve the linked article.